src/views/RecordingPreviewer/RecordingPreviewer.tsx

```diff
@@ -30,6 +31,11 @@ export function RecordingPreviewer() {
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
   const isDiscardable = Boolean(state?.discardable)
   invariant(fileName, 'fileName is required')
+  const file: StudioFile = {
+    fileName,
+    displayName: getFileNameWithoutExtension(fileName),
```

Comment:
Note: this is getting repetitive and is fairly error-prone - `displayName` is always (at least for now) the same as the file name without the extension, so it can be easily derived from it without the need to keep both values in sync.